doc/faq.rst

```diff
@@ -24,7 +24,69 @@ There are a couple of reasons for holding off on this for now:
   best.
 
 That said, we do plan to introduce a simple but effective verification
-mechanism at some point. Stay tuned.
+mechanism at some point.
+
+The `NEXTLEAP EU project <https://nextleap.eu>`_ has done some R&D
+regarding key verification questions and protection against active
+network-adversary attacks, see `the countermitm verification protocols
+<https://countermitm.readthedocs.io/en/latest/new.html>`_ some of which
+are implemented with the Autocrypt-enabled `DeltaChat <https://delta.chat>`_ app.
+DeltaChat implemented "setup contact" and "verified group join" work flows
+which try to tackle key verification without talking to users about keys.
+This has not yet been fully discussed within the Autocrypt context but
+it's an interesting approach to look at as it is implemented already
+and received a first round of thorough user-testing late 2018.
+
+How does Autocrypt differ from PEP?
+---------------------------------------------------------------
+
+Autocrypt and PEP are different approaches to opportunistic e-mail encryption.
+
+The PEP effort predates Autocrypt by a few years and involves a foundation
+and two companies which provide FOSS and paid offerings.  PEP offers the
+GPL-licensed PEP-Engine and adapters which can be integrated into MUAs
+if licensing permits.  Some specification efforts have started but
```

Comment:
Specification efforts had not started when we began work on Autocrypt ([first commit](https://pep.foundation/dev/repos/internet-drafts/rev/48f768143a7c) is from december 2016), and so far there has been no official release.